web/src/unread.js

```diff
@@ -864,9 +876,11 @@ export function initialize(params) {
     unread_pm_counter.set_huddles(unread_msgs.huddles);
     unread_pm_counter.set_pms(unread_msgs.pms);
     unread_topic_counter.set_streams(unread_msgs.streams);
-
     for (const message_id of unread_msgs.mentions) {
         unread_mentions_counter.add(message_id);
+        if (unread_msgs.pms[0]?.unread_message_ids.includes(message_id)) {
```

Comment:
Why is `pms[0]` the right thing to check here? The documentation for https://zulip.com/api/register-queue does not seem to document any special behavior for the first entry there.

Reply:
Thanks, for pointing this out this was the wrong code. Fixed this in the updated PR.